fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -389,6 +390,39 @@ struct GetMappedRangeWorkload : ApiWorkload {
 		}
 	}
 
+	ACTOR static Future<Void> reportMetric(GetMappedRangeWorkload* self, Database cx) {
+		StatusObject result = wait(StatusClient::statusFetcher(cx));
+		StatusObjectReader statusObj(result);
+		StatusObjectReader statusObjCluster;
+		StatusObjectReader processesMap;
+
+		if (!statusObj.get("cluster", statusObjCluster)) {
+			TraceEvent("NoCluster");
+			return Void();
+		}
+
+		long queryQueueMax = 0;
+		if (statusObjCluster.get("processes", processesMap)) {
+			TraceEvent("NoProcesses");
+			return Void();
+		}
+		for (auto proc : processesMap.obj()) {
+			StatusObjectReader process(proc.second);
+			if (process.has("roles")) {
+				StatusArray rolesArray = proc.second.get_obj()["roles"].get_array();
+				for (StatusObjectReader role : rolesArray) {
+					if (role["role"].get_str() == "storage") {
+						role.get("query_queue_max", queryQueueMax);
+						TraceEvent("QueryQueueMax").detail("Value", queryQueueMax);
+					}
+				}
+			} else {
+				TraceEvent("NoRoles");
+			}
+		}
```

Comment:
Before return, can you add an assertion that the max queue length is less than, say 1000?
We probably also want test coverage, like `TEST(max_queue_length > 10)`.

Reply:
will discuss offline

Reply:
please check the sample output in PR description, I have many runs and with the fix it is pretty normal to have < 10 queue length, so I have it as `TEST(queryQueueMax > 0)` here.